.github/workflows/test-integration.yaml

```diff
@@ -54,11 +61,6 @@ jobs:
 
       - name: 🎯 Check cache hit
         run: echo '${{ steps.setup-python.outputs.cache-hit }}'
-      - name: 🌈 Install pangeo-forge-recipes & pangeo-forge-runner
-        shell: bash -l {0}
-        run: |
-          python -m pip install ${{ matrix.runner-version }}
-          python -m pip install -e ".[test,minio]"
```

Comment:
Sorry for the piecemeal review, @norlandrhagen.

Since we're not doing anything different re: pip, can we just preserve this block for a cleaner diff?

I would prefer to _not_ support a user-facing `grib` optional dependency (at least as part of this PR), as I'm concerned that without putting that in a conda environment we can really ensure that will work reliably in a cross-platform way.

So I'd prefer just a separate workflow step here called "install grib deps" or similar, and not to have those deps as part of our pyproject.toml.

Reply:
Seems like a good call!

Reply:
If I understand.. I removed the grib deps from pyproject, reverted the install to`python -m pip install -e ".[test,minio]"` and added another step in the CI to install the grib deps via pip.